src/50_Sample_AMPs/Recipe.html

```diff
@@ -0,0 +1,460 @@
+<!---{
+  "preview": true,
+  "draft": true
+  }--->
+<!-- #### Introduction -->
+<!--
+This is a sample recipe AMP article demonstrating how to express machine-readable recipe data using JSON+LD. It uses AMP components that work well for recipe articles: image gallery (carousel), reviews carousel, social sharing, and related articles. There are also star ratings, read-only for now (see [this GitHub issue](https://github.com/ampproject/amphtml/issues/2691)).
```

Comment:
nit: "It uses AMP components that work well for recipe articles: image gallery (carousel), reviews carousel, social sharing, and related articles. " -> you list features not components. Either replace with actual used components or rephrase to something like: "This article demonstrates how different AMP components can be used to implement features such as image galleries, reviews carousel, social sharing and related articles."